main/settings.py

```diff
@@ -206,21 +206,21 @@
 }
 
 MIDDLEWARE = [
+    'middlewares.cache.UpdateCacheForUserMiddleware',
     'debug_toolbar.middleware.DebugToolbarMiddleware',
     'django.middleware.security.SecurityMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
     'corsheaders.middleware.CorsMiddleware',
     'django.middleware.locale.LocaleMiddleware',
     # 'middlewares.middlewares.LocaleMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
-    # ¤ 'middlewares.cache.UpdateCacheForUserMiddleware',
     'django.middleware.common.CommonMiddleware',
-    # ¤ 'middlewares.cache.FetchFromCacheForUserMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
     'middlewares.middlewares.RequestMiddleware',
     'reversion.middleware.RevisionMiddleware',
+    'middlewares.cache.FetchFromCacheForUserMiddleware',
```

Comment:
We will need to move this back as well, as we will need headers from other middlewares as well.